packages/core/cli/src/scripts/createProject/processMagicComments.ts

```diff
@@ -0,0 +1,29 @@
+import fs from 'fs';
+
+const removeMagicComments = async (absoluteFilePath: string): Promise<void> => {
```

Comment:
`removeProjectOnly` would be more descriptive ;)

Reply:
Are you sure? It is about `// core-development-start & end`. Below we have one from removing `Project only`